working/manual_changes/2020/2020-06-29.sql

```diff
@@ -0,0 +1,23 @@
+--create new Relationship_id for PPI
+DO $_$
+BEGIN
+	PERFORM vocabulary_pack.AddNewRelationship(
+	pRelationship_name			=>'Has question source',
```

Comment:
@Yevhenii-Bezeha 
"source of question" or "question's source"?

Reply:
PPI team gave us the source. They call it - Appendix for each survey. And Question source - it it a name of one column there. So I used this relationship_id  as copy from that column, if I understood you correctly.

Reply:
Git it, but a little bit confusing. I'd prefer more explicit "source of question".
@dimshitc @mik-ohdsi What do you think?